debug_toolbar/static/debug_toolbar/js/redirect.js

```diff
@@ -0,0 +1,3 @@
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
This should be redundant with defer

Reply:
Idk how far back django-debug-toolbar tries to support, but ancient browsers don't support `defer`, so best to leave it here?